cli/src/revset_util.rs

```diff
@@ -107,6 +107,31 @@ impl<'repo> RevsetExpressionEvaluator<'repo> {
     }
 }
 
+fn warn_user_redefined_builtin(
+    ui: &Ui,
+    source: &ConfigSource,
+    name: &str,
+) -> Result<(), CommandError> {
+    match source {
+        ConfigSource::Env | ConfigSource::User | ConfigSource::Repo | ConfigSource::CommandArg => {
+            let checked_mutability_builtins =
+                ["mutable()", "immutable()", "builtin_immutable_heads()"];
```

Comment:
nit: technically, these aliases could be overridden by `mutable(    )`, etc., but it's unlikely, and missed warning isn't problem.

(no action needed)

Reply:
Acknowledged.